hail/src/main/scala/is/hail/lir/X.scala

```diff
@@ -176,13 +176,14 @@ class Method private[lir] (
   def findBlocks(): Blocks = {
     val blocksb = new ArrayBuilder[Block]()
 
-    val s = new mutable.Stack[Block]()
+    var s = List[Block]()
```

Comment:
can we just make this a `new ArrayStack[Block]()`? I saw this a few weeks ago and meant to fix it, so glad this came up again.

Reply:
ArrayStack is a one of ours, is.hail.utils.ArrayStack